### PR TITLE
[Merged by Bors] - Document collide args

### PR DESCRIPTION
**Pull request by rukai**
Fixes https://github.com/bevyengine/bevy/issues/2720#issuecomment-904623168

Possibly we should omit the extra bit in brackets, not sure if its warranted.

### CODE REVIEW

File: crates/bevy_sprite/src/collide_aabb.rs [hunk 1]
```diff
@@ -12,6 +12,8 @@ pub enum Collision {
 
 // TODO: ideally we can remove this once bevy gets a physics system
 /// Axis-aligned bounding box collision with "side" detection
+/// *   a_pos and b_pos are the center positions of the rectangles. (This is different to traditional AABB where the position is the corner)
```
Comment (alice-i-cecile) [inline crates/bevy_sprite/src/collide_aabb.rs:15]:
```suggestion
/// *   a_pos and b_pos are the center positions of the rectangles, typically obtained by extracting the `translation` field from a `Transform` component
```

Comment (alice-i-cecile) [reply crates/bevy_sprite/src/collide_aabb.rs:15]: This adds some more context on how to use this function.

I think the original parenthetical statement is distracting and not terribly helpful once the fields' meaning is documented.

**Comment by rukai** [comment]
I like that suggestion! Makes it very clear how to use.

**Review by alice-i-cecile — APPROVED** [review]

**Review by gark-msft — APPROVED** [review]

**Comment by Weibye** [comment]
Would something like

```Rust
/// # Arguments
/// 
/// * `a_pos` - Center-position of the first rectangle, 
/// * `a_size` - Size of the first rectangle in height; width
/// * `b_pos` - Center-position of the second rectangle
/// * `b_size` - Size of the second rectangle in height; width
```
be too much or would it be useful?

**Comment by cart** [comment]
We don't see "structured" argument docs like that very often in rust docs. Generally they are documented free-form in sentences. I think there are arguments to be made for both approaches, but given that this goes against a pattern we currently use, it should probably be discussed separately.

**Pull request closed without merging**